server/model/picture.js

```diff
@@ -10,10 +10,9 @@ export type Picture = {|
   alt?: ?string;
   copyright?: ?string;
   author?: string;
-  copyrightHolder?: string;
   fileFormat?: string;
   url?: ?string;
-  attribute?: {title: ?string, author: ?string, source: ?string, license: LicenseType};
+  attribute?: {title: ?string, author: ?string, sourceName: ?string, sourceLink: ?string, license: LicenseType, copyrightHolder: ?string, copyrightLink: ?string};
```

Comment:
Can you explain why you wouldn't want this on the root - I can think of many instances where I might just want one bit of this information - it isn't a standardised nor logical grouping, bar for display purposes.